R/deprecated.R

```diff
@@ -58,32 +58,32 @@
 #'
 #' @examples
 #' # A deprecated function `foo`:
-#' deprecate_warn("1.0.0", "foo()")
+#' deprecate_warn("1.0.0", foo())
 #'
 #' # A deprecated argument `arg`:
-#' deprecate_warn("1.0.0", "foo(arg = )")
+#' deprecate_warn("1.0.0", foo(arg))
 #'
 #' # A partially deprecated argument `arg`:
-#' deprecate_warn("1.0.0", "foo(arg = 'must be a scalar integer')")
+#' deprecate_warn("1.0.0", foo(arg = 'must be a scalar integer'))
 #'
 #' # A deprecated function with a function replacement:
-#' deprecate_warn("1.0.0", "foo()", "bar()")
+#' deprecate_warn("1.0.0", foo(), bar())
 #'
 #' # A deprecated function with a function replacement from a
 #' # different package:
-#' deprecate_warn("1.0.0", "foo()", "otherpackage::bar()")
+#' deprecate_warn("1.0.0", foo(), "otherpackage::bar()")
```

Comment:
Still quoted: `"otherpackage::bar()"`

Reply:
Yeah, otherwise R CMD check warns about use of `otherpackage`